src/tm_case.c

```diff
@@ -226,7 +226,7 @@ static const union AnimCmd *const sTMSpriteAnims[] = {
 };
 
 static const struct CompressedSpriteSheet sTMSpriteSheet = {
-    (const void *)gTMCase_TMSpriteGfx,
+    (const void *)gTMCaseDisc_Gfx,
```

Comment:
```suggestion
    gTMCaseDisc_Gfx,
```

Reply:
Just want to acknowledge I've seen this. I started documenting the rest of that file but got sidetracked, I'll return to it at some point